Makefile

```diff
@@ -19,11 +19,11 @@ test:
 test-integration:
 	bundle exec rspec
 
-build-production: test
+build-production:
```

Comment:
If it wasn't for the oauth test, the tests for this thing would execute in a fraction of a second. I wonder if we're better off speeding up that test and leaving this in? Then your binary is much more likely to be correct.

Reply:
Is it more likely? If you're ever shipping something that hasn't passed CI then I think you're in a bad way right? I honestly think with CI in place we don't need to enforce this, and it just makes the development process a lot more smooth. Otherwise I'm just writing `go build <blurgh>` every time, instead of using the makefile.

Reply:
why though? if tests take a fraction of a second you can just `make build-production` right?

Reply:
Also - why do you need to build a linux version when developing locally?

Reply:
Seems to me that it's better to be sure that the binary you build isn't going to immediately crash when you're testing things, and it's ok to wait 0.2 seconds for that.

Reply:
Ok I'll leave it back in

Reply:
Ok I'll leave it back in. It's pretty crap when you just want to rebuild a tiny change, or have just build to production binary and then go to build the client and you re-run exactly the same tests.

I see it kinda like git hooks- you've opted someone into a flow, rather than trusting them to do the right thing. Don't personally agree with that but it's a personal preference.

Reply:
I still don't understand why it's crap. How does this affect your flow at all? (if you imagine that the tests don't take 5s to complete)

Reply:
If the tests don't take 5s then it becomes a lot better, it's just right now when they do it's quite irritating. It also just feels imprecise- I asked it to build but it's testing 🤷‍♂️ 

It's not an issue though- I've reverted the change!

Reply:
To be fair, you asked it to create a production build. If the next step is to package the build an d ship it, it doesn't seem too crazy to enforce that tests pass. I do agree that it's painful right now when the tests take 5 seconds!